jishaku/features/root_command.py

```diff
@@ -150,10 +150,16 @@ async def jsk(self, ctx: commands.Context):
             message_cache = "Message cache is disabled"
 
         if discord.version_info >= (1, 5, 0):
-            presence_intent = f"presence intent is {'enabled' if self.bot.intents.presences else 'disabled'}"
-            members_intent = f"members intent is {'enabled' if self.bot.intents.members else 'disabled'}"
+            intents_info = [
+                f"presence intent is {'enabled' if self.bot.intents.presences else 'disabled'}",
+                f"members intent is {'enabled' if self.bot.intents.members else 'disabled'}"
+            ]
 
-            summary.append(f"{message_cache}, {presence_intent} and {members_intent}.")
+            if hasattr(self.bot.intents, "message_content"):
+                # The privileged message content intent is supported
+                intents_info.append(f"message content intent is {'enabled' if self.bot.intents.message_content else 'disabled'}")
```

Comment:
it would probably make sense to share this information regardless, eg:

> message content intent is unknown


If the attribute exists, then this can be set to enabled/disabled.